### ISSUE TITLE
[BUG] Installing the Doppler CLI inside a docker image sometimes fails with a 500 error

### ISSUE THREAD

**Issue by salaerts** [DopplerHQ/cli#202]
**Describe the bug**
We have a number of Docker images that install the Doppler CLI as part of the image. Since Friday April 16th the image builds have started failing occasionally with the message _"ERROR: Download failed with status 500"_. (To be clear, this is not consistent behavior, sometimes the images builds do succeed.)

**To Reproduce**
Steps to reproduce the behavior. Please include output from running the command with `--debug`.
1. Make a Dockerfile with this content:
```
FROM mcr.microsoft.com/dotnet/runtime-deps:5.0-alpine AS base
RUN apk --no-cache add curl icu-libs

ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false

# Install doppler CLI
RUN (curl -Ls https://cli.doppler.com/install.sh || wget -qO- https://cli.doppler.com/install.sh) | sh

# ... remaining file omitted
```
2. Build the image on Azure Devops using the ubuntu-latest vm pool

**Expected behavior**
The image should build successfully.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 20.04.2

**CLI Version:**
Latest

**Additional context**

**Comment by Piccirello** [comment]
Thanks for reporting this. We're getting reports of this intermittent behavior but haven't been able to pinpoint the exact cause yet.

While we continue to diagnose this, could you try our apk repo? This method is just as fast as the install.sh script and should bypass the route that's returning intermittent 500s. You can find instructions [here](https://github.com/DopplerHQ/cli/blob/master/INSTALL.md#alpine-apk).

**Comment by Piccirello** [comment]
This should be fixed as of 7c5057d581afecae2bc35ddb396ea070c1f66710.
